soundtrack/src/SoundtrackPlugin.cpp

```diff
@@ -2,127 +2,26 @@
 // Copyright Twitch Interactive, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: GPL-2.0
 
-#include <obs-frontend-api.h>
 #include <obs-module.h>
-
-#include "OBSCallback.h"
 #include "Util.h"
 #include "obs-source.h"
 #include <util/config-file.h>
 #include <util/dstr.h>
-#include <optional>
 
 OBS_DECLARE_MODULE()
 
 OBS_MODULE_USE_DEFAULT_LOCALE("soundtrack-plugin", "en-US")
 
 extern struct obs_source_info soundtrack_source;
 
-namespace Twitch::Audio {
-std::optional<OBSCallback> callback;
-static obs_encoder_t *archiveEncoder = nullptr;
-} // namespace Twitch::Audio
-
 bool obs_module_load(void)
 {
     auto minVersion = static_cast<uint32_t>(MAKE_SEMANTIC_VERSION(26, 0, 0));
```

Comment:
Is the SL obs version aligned?

Reply:
Sorry just seeing this now, yes it is aligned.